rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/release/ReleaseController.java

```diff
@@ -1282,6 +1286,34 @@ public ResponseEntity<?> loadSpdxLicensesInfo(
         return new ResponseEntity<>(responseBody, HttpStatus.OK);
     }
 
+    @GetMapping(value = RELEASES_URL + "/{id}/releases")
+    public ResponseEntity<CollectionModel<HalResource<ReleaseLink>>> getLinkedReleases(
+            @PathVariable("id") String id,
+            @RequestParam(value = "transitive", required = false, defaultValue = "false") boolean transitive
+    ) throws TException {
```

Comment:
Please add relevant `@Operation` and `@Parameter` annotations for correctly document the endpoint in OpenAPI doc.